packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -51,7 +51,7 @@ mkdir -p /home/ubuntu/.local/share/lpass
 
 # clone our repo
 cd /home/ubuntu
-git clone https://github.com/beyondessential/tupaia.git
+sudo -Hu ubuntu git clone https://github.com/beyondessential/tupaia.git
```

Comment:
Building new gold master images takes a while so I'm actually still testing this, won't merge until I have confirmed it works (or rolled it back)

Reply:
For future reference @IgorNadj this doesn't seem to have changed the owner of the tupaia repo, but it also did no harm, so I just left it in.